pandas/tests/io/json/test_pandas.py

```diff
@@ -935,6 +935,7 @@ def test_default_handler_numpy_unsupported_dtype(self):
                     '["(4-5j)","(1.2+0j)"]]')
         assert df.to_json(default_handler=str, orient="values") == expected
 
+    @pytest.mark.skip(reason='Trying to find CI failure (TODO: fix me after!)')
```

Comment:
Can you revert this change?

I'd like to merge PR (even if stuff isn't all passing) to get all the boto issues fixed. I'm debugging everything else in https://github.com/pandas-dev/pandas/pull/24092

Reply:
Well, let's just leave it for now since CI is running and I'll follow up.